pyquil/latex/ipython.py

```diff
@@ -0,0 +1,51 @@
+import os
+import subprocess
+import shutil
+import tempfile
```

Comment:
lets rename this file to `_ipython.py` to make it "private" and just import the `display` function to the `latex` module API (as you already do)